### PR TITLE
use a remote config in kuberun

### PR DESCRIPTION
**Pull request by Lehmann-Fabian**
Using kuberun with remote configs is not yet possible. Therefore, I added the option to use a config file only located in the cluster.

### CODE REVIEW

File: modules/nextflow/src/main/groovy/nextflow/cli/CmdKubeRun.groovy [hunk 1]
```diff
@@ -53,6 +53,12 @@ class CmdKubeRun extends CmdRun {
     @Parameter(names = '-head-memory', description = 'Specify amount of memory requested for the Nextflow pod')
     String headMemory
 
+    @Parameter(names=['-rc','-remoteConfig'], description = 'Choose configuration files on remote fs', hidden = true )
+    List<String> runRemoteConfig
```
Comment (bentsherman) [inline modules/nextflow/src/main/groovy/nextflow/cli/CmdKubeRun.groovy:57]: I would leave out the `-rc` shortcut just to keep things simple.

I would try to make the help text consistent with `-config`, e.g. "Add the specified file from the K8s cluster to configuration set"

File: modules/nextflow/src/main/groovy/nextflow/k8s/K8sDriverLauncher.groovy [hunk 2]
```diff
@@ -248,7 +248,7 @@ class K8sDriverLauncher {
                 .setProfile(cmd.profile)
                 .setCmdRun(cmd)
 
-        if( !interactive && !pipelineName.startsWith('/') ) {
+        if( !interactive && !pipelineName.startsWith('/') && !cmd.remoteProfile && !cmd.runRemoteConfig ) {
```
Comment (bentsherman) [inline modules/nextflow/src/main/groovy/nextflow/k8s/K8sDriverLauncher.groovy:251]: Would just checking `cmd.runRemoteConfig` be sufficient here?

**Review by bentsherman — CHANGES_REQUESTED** [review]
Looks good to me overall. Just a few small things and then we'll be ready to merge:
- please address my comments below
- add the new options to the [Nextflow kuberun docs](https://www.nextflow.io/docs/latest/cli.html#kuberun)
- confirm that you've tested this feature on your k8s cluster

File: modules/nextflow/src/main/groovy/nextflow/cli/CmdKubeRun.groovy (hunk 1 above)
Comment (Lehmann-Fabian) [reply modules/nextflow/src/main/groovy/nextflow/cli/CmdKubeRun.groovy:57]: Good point. I addressed it.

**Review by bentsherman — APPROVED** [review]

**Comment by Lehmann-Fabian** [comment]
@bentsherman can you please start the checks manually? The CI Pipeline seems not to start automatically.

**Comment by bentsherman** [comment]
@pditommaso Can you tell why the CI tests aren't running? I can't trigger them manually. It just says "Waiting for status to be reported", I don't know what that means.

**Comment by Lehmann-Fabian** [comment]
I think this PR should fix it: https://github.com/nextflow-io/nextflow/pull/2796

**Pull request merged**